### DISCUSSION TITLE
Release principle: one transition release

### DISCUSSION THREAD

**Discussion by mlutze** [flix/flix#8347] [Ideas]
Based on discussion in https://github.com/flix/flix/issues/4033

We could have some principle that says that for breaking changes, we have one release where both the new and old versions of feature X are usable.

I recognize that this is often complex or not doable, so we can caveat the principle appropriately.

**Comment by magnus-madsen** [comment]
Seems reasonable.

**Comment by JonathanStarup** [comment]
It would make sense to have a label for "do before next release" so we dont forgot to remove the temporary part
